src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -233,6 +237,156 @@ where
         }))
     }
 
+    async fn create_sink(
+        &self,
+        request: Request<CreateSinkRequest>,
+    ) -> Result<Response<CreateSinkResponse>, Status> {
+        self.ddl_lock.read().await;
+        self.env.idle_manager().record_activity();
+
+        let req = request.into_inner();
+        let mut mview = req.get_sink().map_err(tonic_err)?.clone();
```

Comment:
What is `mview` here?

Reply:
`mview` here in `create_sink` is a `SInk` so I've renamed it as such.